superset-frontend/src/explore/components/ExploreChartHeader/index.jsx

```diff
@@ -248,134 +226,142 @@ export class ExploreChartHeader extends React.PureComponent {
-    const queryResponse = queriesResponse?.[0];
-    const chartFinished = ['failed', 'rendered', 'success'].includes(
-      this.props.chart.chartStatus,
-    );
-    return (
-      <StyledHeader id="slice-header" className="panel-title-large">
-        <div className="title-panel">
-          {slice?.certified_by && (
-            <>
-              <CertifiedBadge
-                certifiedBy={slice.certified_by}
-                details={slice.certification_details}
-              />{' '}
-            </>
-          )}
-          <EditableTitle
-            title={this.getSliceName()}
-            canEdit={
-              !this.props.slice ||
-              this.props.can_overwrite ||
-              (this.props.slice?.owners || []).includes(
-                this.props?.user?.userId,
-              )
-            }
-            onSaveTitle={this.props.actions.updateChartTitle}
-          />
+  useEffect(() => {
+    if (canAddReports()) {
+      fetchUISpecificReport(user.userId, 'chart_id', 'charts', chart.id);
+    }
+    if (dashboardId) {
+      fetchChartDashboardData();
+    }
+  }, [
+    canAddReports,
+    fetchUISpecificReport,
+    user.userId,
+    chart.id,
+    dashboardId,
+    fetchChartDashboardData,
+  ]);
 
-          {this.props.slice && (
-            <StyledButtons>
-              {user.userId && (
-                <FaveStar
-                  itemId={this.props.slice.slice_id}
-                  fetchFaveStar={this.props.actions.fetchFaveStar}
-                  saveFaveStar={this.props.actions.saveFaveStar}
-                  isStarred={this.props.isStarred}
-                  showTooltip
-                />
-              )}
-              {this.state.isPropertiesModalOpen && (
-                <PropertiesModal
-                  show={this.state.isPropertiesModalOpen}
-                  onHide={this.closePropertiesModal}
-                  onSave={this.props.sliceUpdated}
-                  slice={this.props.slice}
-                />
-              )}
-              <Tooltip
-                id="edit-desc-tooltip"
-                title={t('Edit chart properties')}
+  const {
+    chartStatus,
+    chartUpdateEndTime,
+    chartUpdateStartTime,
+    latestQueryFormData,
+    queriesResponse,
+  } = chart;
+  // TODO: when will get appropriate design for multi queries use all results and not first only
+  const queryResponse = queriesResponse?.[0];
+  const chartFinished = ['failed', 'rendered', 'success'].includes(
+    chart.chartStatus,
+  );
+  return (
+    <StyledHeader id="slice-header" className="panel-title-large">
+      <div className="title-panel">
+        {slice?.certified_by && (
+          <>
+            <CertifiedBadge
+              certifiedBy={slice.certified_by}
+              details={slice.certification_details}
+            />{' '}
+          </>
+        )}
+        <EditableTitle
+          title={getSliceName()}
```

Comment:
```suggestion
          title={getSliceName}
```
This doesn't need to be instantiated.